internal/commands/push.go

```diff
@@ -34,9 +34,10 @@ type pushOptions struct {
 func pushCmd(dockerCli command.Cli) *cobra.Command {
 	var opts pushOptions
 	cmd := &cobra.Command{
-		Use:   "push [<app-name>]",
-		Short: "Push the application to a registry",
-		Args:  cli.RequiresMaxArgs(1),
+		Use:     "push [APP_NAME] --tag TARGET_REFERENCE [OPTIONS]",
+		Short:   "Push an application package to a registry",
```

Comment:
I wonder if we should document the fact that Push can be used to relocate a bundle (e.g.: `docker app push sourcerepo/app:tag --tag my-internal-registry.local:5000/prod/my-app:tag`)

Reply:
Not in the CLI help but it should be documented, yes.

@silvin-lubecki also has some ideas about how to bring the current bundle command closer to what we have for `docker build` so we need to see how it fits with that.